src/formats/XYZ.cpp

```diff
@@ -90,10 +92,10 @@ std::streampos XYZFormat::forward() {
 
     try {
         file_->skiplines(natoms + 1);
-    } catch (const FileError& e) {
+    } catch (const FileError&) {
         // We could not read the lines from the file
         throw format_error(
-            "not enough lines for XYZ format: {}", file_->path(), e.what()
+            "not enough lines for XYZ format in '{}'", file_->path()
```

Comment:
Consider rewriting to *XYZ format: not enough lines* to ensure consistency with other error messages.